brkt_cli/aws/aws_service.py

```
@@ -387,6 +392,14 @@ def get_volumes(self, tag_key=None, tag_value=None):
         get_all_volumes = self.retry(self.conn.get_all_volumes)
         return get_all_volumes(filters=filters)
 
+    def iam_role_exists(self, role):
+        conn = boto.iam.connect_to_region(self.region)
+        try:
+            conn.get_instance_profile(role)
+        except:
```

Do we really want to silently fail here?